aggregator/src/main/scala/weco/concepts/aggregator/LambdaMain.scala

```diff
@@ -0,0 +1,54 @@
+package weco.concepts.aggregator
+
+import com.amazonaws.services.lambda.runtime.{Context, RequestHandler}
+import grizzled.slf4j.Logging
+import weco.concepts.aggregator.sources.WorksSnapshotSource
+
+import java.util.{Map => JavaMap}
+import scala.concurrent.Await
+import scala.concurrent.duration.DurationInt
+
+object LambdaMain
+    extends RequestHandler[JavaMap[String, String], String]
+    with AggregatorMain
+    with Logging {
+
+  override def handleRequest(
+    event: JavaMap[String, String],
+    context: Context
+  ): String = {
+    val workId = event.get("workId")
```

Comment:
The message format is not settled and the logic for choosing whether to fetch the snapshot or an individual Work is a bit of a placeholder.

I suspect this should be two distinct lambdas.  One to fetch it all, and one to fetch one work.  That way we could have distinct memory and timeout settings for them

Reply:
Very much inclined to agree re separate Lambdas